dials_data/definitions/mpro_x0692.yml

```
@@ -0,0 +1,416 @@
+name: mpro-x0692
+author: DLS I04-1/XChem (2020)
+license: CC-BY 4.0
+url: https://zenodo.org/record/3730940#.YQl3gFNKiAk
```

```suggestion
url: https://doi.org/10.5281/zenodo.3730939
```
We usually use the DOI link here